packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as vscode from 'vscode';
+import { DebugConfigurationProvider } from './adapter/debugConfigurationProvider';
+import { nls } from './messages';
+
+export function activate(context: vscode.ExtensionContext) {
+  console.log('Apex Replay Debugger Extension Activated');
+  context.subscriptions.push(
+    vscode.commands.registerCommand(
+      'extension.replay-debugger.getLogFileName',
+      config => {
+        return vscode.window.showInputBox({
+          placeHolder: nls.localize('prompt_for_log_file_text')
```

Comment:
@DatGuyJonathan I think it would be more convenient for the user if we prompt the user with showOpenDialog so that they interact with the native file explorer to find the logs. Would save the user some steps in getting the full file path since this doesn't provide autocomplete and if they do have it in their clipboard, it would be better to put it in the launch.json. Thoughts?

Reply:
It's an interesting idea. We're going to have a command to download debug logs, but it'll default to the hidden .sfdx folder. Could you walk me through some a flow that involves selecting a hidden file with showOpenDialog?

Reply:
@DatGuyJonathan I had the chance to play with this quickly:

```
  context.subscriptions.push(
    vscode.commands.registerCommand(
      'extension.replay-debugger.getLogFileName',
      config => {
        return vscode.window.showOpenDialog({
          canSelectFiles: true,
          canSelectFolders: false,
          canSelectMany: false,
          openLabel: nls.localize('prompt_for_log_file_text') // This is a bit ugly in the GIF
        });
      }
    )
  );
```

![opendialog](https://user-images.githubusercontent.com/54414/34744247-7161b0de-f541-11e7-8869-898e030f5bde.gif)

The API is at https://code.visualstudio.com/docs/extensionAPI/vscode-api#OpenDialogOptions

I think we can use the OpenDialog and default the folder to the .sfdx/logs folder. The OpenDialog is a bit more general and can allow the user to select a different log.

The downside is that the user might select a log file that doesn't have any source that belongs to the current workspace.